Sources/MapboxNavigation/NavigationMapView.swift

```diff
@@ -1760,21 +1716,113 @@ open class NavigationMapView: UIView {
                 symbolLayer.textOpacity = .constant(0.75)
                 symbolLayer.textAnchor = .constant(.bottom)
                 symbolLayer.textJustify = .constant(.left)
-                try mapView.mapboxMap.style.addPersistentLayer(symbolLayer)
+                let layerPosition = addLayerPosition(for: NavigationMapView.LayerIdentifier.voiceInstructionLabelLayer)
+                try mapView.mapboxMap.style.addPersistentLayer(symbolLayer, layerPosition: layerPosition)
                 
                 var circleLayer = CircleLayer(id: NavigationMapView.LayerIdentifier.voiceInstructionCircleLayer)
                 circleLayer.source = NavigationMapView.SourceIdentifier.voiceInstructionSource
                 circleLayer.circleRadius = .constant(5)
                 circleLayer.circleOpacity = .constant(0.75)
                 circleLayer.circleColor = .constant(.init(.white))
-                try mapView.mapboxMap.style.addPersistentLayer(circleLayer)
+                try mapView.mapboxMap.style.addPersistentLayer(circleLayer, layerPosition: .above(NavigationMapView.LayerIdentifier.voiceInstructionLabelLayer))
             }
         } catch {
             Log.error("Failed to perform operation while adding voice instructions with error: \(error.localizedDescription).",
                       category: .navigationUI)
         }
     }
     
+    func addLayerPosition(for layerIdentifier: String, route: Route? = nil, customLayerPosition: MapboxMaps.LayerPosition? = nil) -> MapboxMaps.LayerPosition? {
+        guard customLayerPosition == nil else { return customLayerPosition }
+        
+        let belowSymbolLayers: [String] = [
```

Comment:
I think that maneuver arrow should be shown above road name, in some of the cases it's almost hidden:

![Screen Shot 2022-08-02 at 11 01 25 AM](https://user-images.githubusercontent.com/1496498/182444451-52ef5bb0-7648-44d2-a006-7a6dc3b41da4.png)

Reply:
I see. Thanks for the suggestion, I'll add the customized layer position of the arrow layer as above the layer that contains `road-label`. Just tested it with the test App, it seems that all the styles contain the layer for `road-label`. It should work. If it's not found, it will be below the symbol layer.

Reply:
I checked the layer positions in different styles. The road name label is always below the exit shield or other road symbols. So it's safe to place the arrow layers just above the road name label when it exits. The new commit fixed this issue.

Reply:
Its fixed in the new commits with test added.